Modules/ModParser-3_0.lua

```diff
@@ -2029,6 +2032,7 @@ local specialModList = {
 	["you have resolute technique while you do not have elemental overload"] = { mod("Keystone", "LIST", "Resolute Technique", { neg = true, type = "Condition", var = "HaveElementalOverload" }) },
 	["hits ignore enemy monster fire resistance while you are ignited"] = { flag("IgnoreFireResistance", { type = "Condition", var = "Ignited" }) },
 	["your hits can't be evaded by blinded enemies"] = { flag("CannotBeEvaded", { type = "ActorCondition", actor = "enemy", var = "Blinded" }) },
+	["blind does not effect your chance to hit"] = { flag("IgnoreBlindHitChance") },
```

Comment:
One wrong word here - "effect" should be "affect".

Reply:
My bad, I copied it straight from the spreadsheet though